### DISCUSSION TITLE
Ne zaman hangi borsa ?

### DISCUSSION THREAD

**Discussion by umitkucuk17** [IBAX-io/go-ibax#1856] [Ideas]
Ne zaman işlem görmeye başlayacak hangi borsalarda işlem görecek bilgisi olan var mı

**Comment by muhmmedcelik** [comment]
https://join.swissborg.com/en/r/muhammFLNS

Üye ol kyc yap 50€ yatır çek biletini ac 100€ kadar hediyeni kap.
